File: gaia-sdk-python/gaia_sdk/graphql/request/input/CancelSkillBuildJobImpulse.py


from typing import Callable, List
from gaia_sdk.api.VariableRegistry import VariableRegistry
from gaia_sdk.graphql.request.enumeration.Order import Order
from gaia_sdk.graphql.request.enumeration.OrderByField import OrderByField
from gaia_sdk.graphql.request.enumeration.EdgeOrderByField import EdgeOrderByField
from gaia_sdk.graphql.request.enumeration.EdgeType import EdgeType


class CancelSkillBuildJobImpulse():
    tenantId: str
    buildJobRef: str

    def __init__(self, tenantId: str, buildJobRef: str):
        self.tenantId = tenantId
        self.buildJobRef = buildJobRef

    def __eq__(self, other):
        if type(other) is type(self):
            return self.tenantId == other.tenantId and self.buildJobRef == other.buildJobRef
        return False

    def __repr__(self):
        return {'tenantId': self.tenantId, 'buildJobRef': self.buildJobRef}

File: gaia-sdk-javascript/src/graphql/request/input/CancelSkillBuildJobImpulse.ts


import VariableRegistry from "../../../api/VariableRegistry"
import {Uuid, ISO8601, Struct} from "../../GaiaClient";
import {Order} from "../enumeration/Order";
import {OrderByField} from "../enumeration/OrderByField";
import {EdgeOrderByField} from "../enumeration/EdgeOrderByField";
import {EdgeType} from "../enumeration/EdgeType";

export class CancelSkillBuildJobImpulse {
public _typeName = "CancelSkillBuildJobImpulse";
    private tenantId:Uuid;
    private buildJobRef:Uuid;

    constructor (tenantId:Uuid, buildJobRef:Uuid) {
        this.tenantId = tenantId;
        this.buildJobRef = buildJobRef;
    }
}
